Amazon.QLDB.Driver.IntegrationTests/utils/IntegrationTestBase.cs

```diff
@@ -192,4 +228,205 @@ private DescribeLedgerResponse DescribeLedger(DescribeLedgerRequest ledgerReques
+            ionTimestampWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionTimestampWithAnnotation });
+
+            var ionNullBlobWithAnnotation = ValueFactory.NewNullBlob();
+            ionNullBlobWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullBlobWithAnnotation });
+
+            var ionNullBoolWithAnnotation = ValueFactory.NewNullBool();
+            ionNullBoolWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullBoolWithAnnotation });
+
+            var ionNullClobWithAnnotation = ValueFactory.NewNullClob();
+            ionNullClobWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullClobWithAnnotation });
+
+            var ionNullDecimalWithAnnotation = ValueFactory.NewNullDecimal();
+            ionNullDecimalWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullDecimalWithAnnotation });
+
+            var ionNullFloatWithAnnotation = ValueFactory.NewNullFloat();
+            ionNullFloatWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullFloatWithAnnotation });
+
+            var ionNullIntWithAnnotation = ValueFactory.NewNullInt();
+            ionNullIntWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullIntWithAnnotation });
+
+            var ionNullListWithAnnotation = ValueFactory.NewNullList();
+            ionNullListWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullListWithAnnotation });
+
+            var ionNullSexpWithAnnotation = ValueFactory.NewNullSexp();
+            ionNullSexpWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullSexpWithAnnotation });
+
+            var ionNullStringWithAnnotation = ValueFactory.NewNullString();
+            ionNullStringWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullStringWithAnnotation });
+
+            var ionNullStructWithAnnotation = ValueFactory.NewNullStruct();
+            ionNullStructWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullStructWithAnnotation });
+
+            var ionNullSymbolWithAnnotation = ValueFactory.NewNullSymbol();
+            ionNullSymbolWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullSymbolWithAnnotation });
+
+            var ionNullTimestampWithAnnotation = ValueFactory.NewNullTimestamp();
+            ionNullTimestampWithAnnotation.AddTypeAnnotation("annotation");
+            ionValues.Add(new object[] { ionNullTimestampWithAnnotation });
+
+            return ionValues;
+        }
+
+        public string GetDisplayName(MethodInfo methodInfo, object[] data)
```

Comment:
Are we using this anywhere?

Reply:
According to this doc: https://dev.to/frannsoft/mstest-v2---new-old-kid-on-the-block
It says: ITestDataSource.GetDisplayName will be displayed when the test is executed via the target runner (VSTest if running in Visual Studio), allowing us to determine which cases of a test failed when tests are parameterized.

Reply:
Ok thanks!